bodhi/client/__init__.py

```diff
@@ -924,6 +924,36 @@ def waive(update: str, show: bool, test: typing.Iterable[str], comment: str, url
         print_resp(resp, client)
 
 
+@updates.command()
+@click.argument('update')
+@openid_option
+@staging_option
+@url_option
+@debug_option
+@handle_errors
+def trigger(update: str, url: str, openid_api: str, **kwargs):
```

Comment:
I don't find trigger to be very explicit on what this command does. As a user typing `bodhi update trigger`does not really speek to me.
Maybe something like `bodhi update trigger-tests` or `bodhi update start-tests` would be better. What do you think ?

Reply:
I wasn't sure how to use option that has two words in it. Looked through bodhi commands and didn't found any. This is why I went for one word option.

Reply:
I think this can be done has suggested above.